share/zsh/site-functions/_pot

```diff
@@ -27,7 +27,7 @@ _pot_flavours() {
 }
 
 _pot_attributes() {
-	_values "pot attributes" 'start-at-boot' 'early-start-at-boot' 'persistent' 'no-rc-script' 'prunable' 'localhost-tunnel' 'enforce_statfs' 'mount' 'fdescfs' 'linprocfs' 'nullfs' 'procfs' 'tmpfs' 'zfs' 'children'
+	_values "pot attributes" 'start-at-boot' 'early-start-at-boot' 'persistent' 'no-rc-script' 'prunable' 'localhost-tunnel' 'no-tmps' 'no-etc-hosts' 'enforce_statfs' 'mount' 'fdescfs' 'linprocfs' 'nullfs' 'procfs' 'tmpfs' 'zfs' 'children'
```

Comment:
no-tmps => no-tmpfs

Reply:
Related: Is enforce_statfs something that should be changed to enforce-statfs at some point to match other attributes?

Reply:
nice catch.
`enforce_statfs` is inherited from `jail`, that's why it uses the `_`.
I don't have a strong opinion for or against `-` and `_`
Adopting only one separator would probably show a higher quality